server/lyft/temporal/cmd/server.go

```diff
@@ -0,0 +1,148 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/runatlantis/atlantis/server/lyft/temporal/workflows"
+	"github.com/spf13/cobra"
+	"go.temporal.io/sdk/client"
+)
+
+func NewServerCmd() *cobra.Command {
+	c := &cobra.Command{
+		Use:           "application-server",
+		Short:         "Start the application server",
+		Long:          `Start the atlantis temporal application worker`,
+		SilenceErrors: true,
+		SilenceUsage:  true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			s := NewServer()
+
+			if err := s.server.ListenAndServe(); err != nil {
+				panic(err)
```

Comment:
nit: let's take advantage of RunE accepting an error return and just pass forward the error rather than panicking.